views/partials/head.ecr

```diff
@@ -10,3 +10,6 @@
 <link rel="apple-touch-icon" sizes="180x180" href="img/favicon.png">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <meta name="google" content="notranslate">
+<%- {% unless flag?(:release) %} -%>
+  <script src="http://localhost:35729/livereload.js"></script>
```

Comment:
Can the liveview server be started and integrated in Lavinmqs http server?

Reply:
Yes

Reply:
... but, I don't think it's worth the effort? You could perhaps set another http server, setup the websocket handler, keep tabs of connection and handle the js yourself. https://github.com/bcardiff/live_reload.cr/blob/master/src/live_reload.cr#L140

But running a separate server like this seems to the standard thing to do https://github.com/livereload/livereload-js?tab=readme-ov-file#what-is-livereload